kumascript/macros/LiveSampleURL.ejs

```diff
@@ -22,7 +22,7 @@ const samplePath = $1 ?
 
 let url;
 if (env.live_samples.base_url.includes('://')) {
-    url = new URL(env.live_samples.base_url);
+    url = new URL($1 ? env.live_samples.legacy_url : env.live_samples.base_url);
```

Comment:
Can we move this into the if-block at line 13?

```js
let base_url = env.live_samples.base_url;
if ($1 && $1.length > 0) {
    pagePath = $1;
    base_url = env.live_samples.legacy_url;
}
```

Reply:
sure